src/providers/realtime-provider.js

```diff
@@ -75,6 +76,29 @@ export default class RealtimeProvider {
         };
     }
 
+    subscribeToLimits(domainObject, callback) {
+        const qualifiedName = idToQualifiedName(domainObject.identifier.key);
+
+        let subscriptionDetails = this.buildSubscriptionDetails(domainObject, callback);
+        let id = subscriptionDetails.subscriptionId;
+        this.subscriptionsById[id] = subscriptionDetails;
+
+        if (this.connected) {
+            let message = MESSAGES.SUBSCRIBE['yamcs.mdbchanges'](subscriptionDetails);
```

Comment:
you can swap this one out too

Reply:
I didn't make it an `OBJECT_TYPE`. I made a new const called `MDB_TYPE`